lib/Dialect/FIRRTL/Transforms/GrandCentralTaps.cpp

```diff
@@ -420,6 +437,8 @@ LogicalResult static applyNoBlackBoxStyleDataTaps(const AnnoPathValue &target,
       // Instance port cannot be used as an annotation target, so use a NodeOp.
       auto node = builder.create<NodeOp>(lastInst.getType(portNo),
                                          lastInst.getResult(portNo));
+      if (loc.isa<FileLineColLoc>())
+        node->setLoc(loc);
```

Comment:
This is really cool that it gets all the way into the generated node! 👍 

Would it be simpler to blindly set the location for all of these as opposed to conditionally updating it if it is a `FileLineColLoc` and not an `UnknownLoc`?